plugin.d.ts

```diff
@@ -31,6 +31,7 @@ declare module 'fastify' {
     sameSite?: boolean | 'lax' | 'strict';
     secure?: boolean;
     signed?: boolean;
+    secret?: string;
```

Comment:
I'm not sure this is correct, it should be added to `unsignCookie`.. but it is missing from this file!

Moreover, we need to update [this](https://github.com/fastify/fastify-cookie/blob/master/test/types.test.ts) accordingly